.github/bin/build_docs.sh

```diff
@@ -36,12 +36,12 @@ TAR_NAME=${REF_VERSION_NAME##*/}
 
 if [[ "$REF_VERSION_NAME" == *"-tm"* ]] ;then
     DIR_VERSION_NAME=$(echo "$TAR_NAME" | sed -r "s/v*(.*)\.tar\.gz/\1/g")
-    TAR_DIR_NAME=incubator-celeborn-$DOC_LINK_PATH
-    wget "https://github.com/apache/incubator-celeborn/releases/download/${REF_VERSION_NAME}"
+    TAR_DIR_NAME=celeborn-$DOC_LINK_PATH
+    wget "https://github.com/apache/celeborn/releases/download/${REF_VERSION_NAME}"
```

Comment:
there is hacky for `-tm` artifacts, we should either keep it, or update the GitHub Release artifacts in the main repo. (I prefer the latter)

Reply:
This could fix in #43.